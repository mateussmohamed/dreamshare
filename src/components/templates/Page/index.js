import React from 'react'
import PropTypes from 'prop-types'
import styled from 'styled-components'

import { Footer } from 'organisms'

const Wrapper = styled.div`
  display: block;
`

const Page = ({ children }) => (
  <Wrapper>
    {children}
    <Footer />
  </Wrapper>
)

Page.propTypes = {
  children: PropTypes.node,
}

export default Page
