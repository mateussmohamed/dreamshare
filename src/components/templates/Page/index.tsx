import React from 'react'

import PropTypes from 'prop-types'
import styled from 'styled-components'

import { Footer } from '../../organisms'

const Wrapper = styled.div`
  display: block;
`

export const Page = props => (
  <Wrapper>
    {props.children}
    <Footer />
  </Wrapper>
)

Page.propTypes = {
  children: PropTypes.node,
}
