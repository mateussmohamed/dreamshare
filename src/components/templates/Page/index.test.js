import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import Page from '.'

describe('[<Page />]', () => {
  const wrap = (props = {}) => shallow(<Page {...props} />).dive()

  test('matches to snapshot', () => {
    const wrapper = wrap({ children: '' })

    expect(wrapper).toMatchSnapshot()
  })
})
