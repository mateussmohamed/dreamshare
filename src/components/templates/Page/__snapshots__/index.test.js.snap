// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<Page />] matches to snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "Page__Wrapper-sc-16vaizz-0",
        "isStatic": true,
        "rules": Array [
          "display:block;",
        ],
      },
      "displayName": "Page__Wrapper",
      "render": [Function],
      "styledComponentId": "Page__Wrapper-sc-16vaizz-0",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  <Footer />
</StyledComponent>
`;
